g2p/convert.py

```diff
@@ -17,14 +17,24 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    __sis_hash_exclude = {"include_pronunciation_variants": False}
-
-    def __init__(self, bliss_lexicon, include_pronunciation_variants=False):
+    __sis_hash_exclude = {
+        "include_orthography_variants": False,
+        "include_pronunciation_variants": False,
+    }
+
+    def __init__(
+        self,
+        bliss_lexicon,
+        include_orthography_variants=False,
```

Comment:
please add new parameters at the end, some people might use non-named arguments.

Reply:
Done.